themes/default/content/blog/ms-teams-integration/index.md

```diff
@@ -0,0 +1,338 @@
+---
+title: "Announcing Microsoft Teams Webhook Integration for Pulumi Cloud"
+allow_long_title: true
+date: 2023-08-28
+meta_desc: Pulumi Cloud now has an easy to set up Microsoft Teas webhooks integration.
+
+meta_image: meta.png
+authors:
+    - meagan-cojocar
+    - komal-ali
+tags:
+    - features
+    - pulumi-cloud
+---
+
+Getting your Pulumi notifications into Microsft Teams is now easier than ever.
+
+<!--more-->
+
+Teams using Pulumi Cloud have been setting up Microsft Teams notifications for their stacks using Pulumi Cloud Webhooks. Webhooks can attach to a Pulumi organization or a Pulumi stack. Starting today, customers can set up a Microsoft Teams integration, for organization or stack notifications, with fewer steps and without needing to host the infrastructure themselves.
+
+Pulumi Cloud Webhooks, including the Microsoft Teams integration, are available to all Pulumi Cloud organizations.
+
+## See it in action!
+
+![Example of the notifications in Microsoft Teams](ms_teams.png)
+
+Pulumi Microsft Teams notifications enable central visibility for your team. In addition to getting the notification of a successful or failed update, you will have links to take you directly to the stack and directly to the update itself. You can see what Pulumi operation was being run, which user initiated it and the number of resouces changed and unaffected. All of this information in one place lets you spend less time context switching and more time collaborating.
+
+### Setting up the Microsoft Teams Integration
+
+Before today Pulumi customers used Pulumi Cloud Webhooks to set up generic JSON webhooks. When events occur, we send a HTTP POST request to any registered listeners. Webhooks can then be used to send notifications to an app (like Microsoft Teams), start running automated tests, or even to update another stack! We have now built support for Microsft Teams formatted webhooks, which allow you to quickly enable notifications about your Pulumi stacks and organizations into your Microsft Teams workspace by simply providing a [incoming webhook URL](https://learn.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook).
```

Comment:
Don't we have a blog post that talks about using webhooks to update another stack? It might be worth adding a link here if so.